README.md

```diff
@@ -44,6 +44,37 @@ For.Net applications, applying the nuget package will automatically add the tele
   </TelemetryModules>
 </ApplicationInsights>
 ```
+
+#### OWIN applications: Adding support to requests, dependencies and live stream
+In order to have requests and dependencies showing in Application Insights Live Stream in **OWIN** applications there are a few additional steps required:

```

Comment:
It may be good to add a link to the post that you mentioned:
http://apmtips.com/blog/2017/02/13/enable-application-insights-live-metrics-from-code

Reply:
I've added a link to the post. Important to notice is that in this case there was not need to connect the module and processor via code. The changes on ApplicationInsights.config were enough.